### DISCUSSION TITLE
Justify text in blog post

### DISCUSSION THREAD

**Discussion by irabanillo91** [adityatelange/hugo-PaperMod#1518] [Q&A]
Is there a way to justify the text, so it goes from looking like 
![image](https://github.com/adityatelange/hugo-PaperMod/assets/60658860/361fbf25-048b-4abc-a5cc-2ce5674ebf3c)
to looking like this
![image](https://github.com/adityatelange/hugo-PaperMod/assets/60658860/8241a9ae-fbdc-4146-9f90-8da1a774373f)

I tried the general solution explained [here](https://github.com/rhazdon/hugo-theme-hello-friend-ng/issues/20) but didn't manage to make it work with papermod.

Thanks a lot!

**Comment by balajijegan** [comment]
Okay, I struggled with this and finally found out what was happening. papermod seems to be wrapping paragraphs `<p>` directly under `<main>`, changing the custom css as below solved the problem for me:

```
main p {
  text-align: justify;
  text-justify: inter-word;
  hyphens: auto;
}
```
